### ISSUE TITLE
Up/down speed indicators are hidden at certain viewport sizes

### ISSUE THREAD

**Issue by Woebin** [Chocobozzz/PeerTube#7134]
### Describe the current behavior

This is a UX issue and an improvement suggestion rather than an outright accessibility violation, but relates most closely to the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 1.4.10 (AA) Reflow](https://www.w3.org/WAI/WCAG22/Understanding/reflow).

The video player widget has up/down arrow icons along with numbers to indicate the current up/down transfer speeds, but at viewport widths between 1420px and 1196px* these disappear entirely. There is a video demonstrating this, but uploading and attaching it to the issue doesn't seem to be working.

*This was observed on a page with both the video player and chat extension embedded, and it's possible that the threshold widths differ depending on the parent page setup. The core issue is that the video player hides these elements when it has a certain amount of width to render within.

### Steps to reproduce

1. On desktop, go to a page with the video player embedded.
2. Make the viewport smaller, for example by resizing the browser window, and watch the up/down speed indicators.
3. Notice the indicators disappearing when you reach a certain viewport width threshold.
4. Continue making the viewport more narrow.
5. Notice the indicators reappearing as you reach another width threshold.

### Describe the expected behavior

WCAG Success Criterion 1.4.10 Reflow only requires content to work at a width of 320px, so this isn't technically a violation.

However, it is removing content for a subset of users, with no way for them to access it. While not a WCAG violation, this makes for an inequitable user experience. We would recommend making the up/down speed indicators available regardless of viewport width, or consider if showing them by default in the first place adds any value for users at all. 

### Additional information

* PeerTube instance:
  * URL: https://peertube-eu.howlround.com/
  * Version: 7.2.1
  * NodeJS version: Unknown
  * Ffmpeg version: Unknown

* Browser name, version and platforms on which you could reproduce the bug: MacOS 15.5, Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).

**Comment by Chocobozzz** [comment]
Hi,

P2P information is a nice have but not more important than player buttons (play/pause, audio, settings, etc.). It's the reason why we choose to hide it when the player is resized. I'm not sure to know how we can display this information on viewport with a reduced width?

**Comment by Chocobozzz** [comment]
Ping @Woebin?

**Comment by Woebin** [comment]
@Chocobozzz apologies, I didn't get a notification about your comments and only got one when you closed the issue.

If you want to improve this based on the original issue (as mentioned, this isn't a full-on violation to begin with), I think one option might be to hide the P2P information on all display sizes and let users toggle it - I may be wrong, but I think to most users this isn't particularly interesting information when watching a live broadcast.

The problem as I see it isn't just that the information is hidden on narrow viewports, but that it's rendered completely unavailable. If the information is meant to be available to users, it should be available to users at all viewport sizes - perhaps overlaid over the video on demand, similar to the settings menu?

**Comment by Chocobozzz** [comment]
No problem and thank you for your answer!

We're developing another player theme where P2P information is hidden on all display sizes. Instead, we injected this information in the "Stats for nerds" overlay. So I think these decisions regarding the new theme address your UX concerns!

**Comment by Woebin** [comment]
Oh that sounds great, the "Stats for nerds" overlay is exactly where I'd personally put it 🙂
